jest.int.config.json

```diff
@@ -0,0 +1,11 @@
+{
+  "setupFiles": [
+    "<rootDir>/src/util/globals.js"
+  ],
+  "moduleFileExtensions": [
+    "js",
+    "jsx"
+  ],
+  "testRegex": "tests_integration\/.*\\.int\\.jsx?$"
```

Comment:
i don't _think_ we would ever have other types of tests, but `tests/integration/` works for me! Will update and merge unless you wanted to look at anything else